src/main/scala/mesosphere/marathon/core/instance/update/InstanceUpdateOpResolver.scala

```diff
@@ -46,7 +46,7 @@ private[marathon] class InstanceUpdateOpResolver(
       case op: GoalChange =>
         updateExistingInstance(op.instanceId)(i => {
           val updatedInstance = i.copy(state = i.state.copy(goal = op.goal))
-          val events = InstanceChangedEventsGenerator.events(updatedInstance, task = None, clock.now(), previousCondition = Some(i.state.condition))
+          InstanceChangedEventsGenerator.events(updatedInstance, task = None, clock.now(), previousCondition = Some(i.state.condition))
```

Comment:
I think this is a bug, the events should be passed to the expression on l.52 or this whole line can be removed - right now no events are generated anyway so it does not matter...

Reply:
File an issue? :)

Reply:
remove it or don't touch this, please :) because the code after the change does not make much sense (as it did not before, it's not really an improvement :) we just got rid of warning which was actually legit here)

Reply:
Okay :) This is the problem with so many warnings. Valuable ones are lost in the noise.